### PR TITLE
Add config for the Stale bot

### PR DESCRIPTION
**Pull request by DokaRus**
Considering [this](https://trello.com/c/LF0wqTHV) card we can use the [Stale bot](https://github.com/probot/stale) to close stale issues and pull requests in our repository.

[Github Marketplace page](https://github.com/marketplace/stale)

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+# type/discussion - our discussion pages. They can be inactive for a long time.
+# WIP - pull requests that have "work in progress" status
+exemptLabels:
+  - type/discussion
+  - WIP
+
+# Label to use when marking as stale
+staleLabel: wontfix
```
Comment (AlekseyMartynov) [inline .github/stale.yml]: I suggest that we introduce a distinct label `stale` for this purpose.

**Review by AlekseyMartynov — DISMISSED** [review]

**Comment by San4es** [comment]
I have some questions:
- Do we need to close [issues with the `type/enhancement` label](https://github.com/DevExpress/DevExtreme/issues?q=is%3Aissue+is%3Aopen+label%3Atype%2Fenhancement) without implementation?
- How to avoid closing an issue (besides the 'type/discussion' label)? For instance, if we have it in our roadmap, like [this issue](https://github.com/DevExpress/DevExtreme/issues/6562#issuecomment-454708703).

/cc @dxbykov @artem-kurchenko

**Comment by AlekseyMartynov** [comment]
> Do we need to close issues with the `type/enhancement` label without implementation?

For this kind of tickets, it makes sense to set a longer `daysUntilStale`, if possible. One year, for example. It will naturally address the second point. If the ticket is inactive for so long then it is clearly not on the roadmap. Which means stale.

**Comment by San4es** [comment]
>set a longer daysUntilStale, if possible. One year, for example.

As far as we form our roadmap yearly, this suggestion sounds good to me.

**Comment by San4es** [comment]
@DokaRus 

Сan we have 2 different strategies - one for issues with the `type/enhancement` label (daysUntilStale = 365) and another one for other issues (daysUntilStale = 30)?

**Comment by DokaRus** [comment]
> @DokaRus
> 
> Сan we have 2 different strategies - one for issues with the `type/enhancement` label (daysUntilStale = 365) and another one for other issues (daysUntilStale = 30)?

[Nope](https://github.com/probot/stale/issues/65)

> Do we need to close issues with the type/enhancement label without implementation?

My point - the team should process such issues shortly. We should decide whether we are going to do it in the long or short term(and prepare a related `type/discussion` in this case), or whether there are more pressing issues at the moment.

>How to avoid closing an issue (besides the 'type/discussion' label)? For instance, if we have it in our roadmap, like this issue.

We can add an one more label to the `exemptLabels` or ignore issues with an assignee

**Comment by San4es** [comment]
>Сan we have 2 different strategies - one for issues with the type/enhancement label (daysUntilStale = 365) and another one for other issues (daysUntilStale = 30)?

>Nope

FYI, looks like GitHub Actions can help implement this workflow:
https://github.com/actions/stale

**Comment by DokaRus** [comment]
>FYI, looks like GitHub Actions can help implement this workflow

Yep. We can start using simple workflow (by adding a configuration for the bot and turning it on).
If our needs will exceed the bot's capabilities, we can move to a more advanced option [custom bot or GitHub actions].

**Comment by San4es** [comment]
>We can add an one more label to the exemptLabels or ignore issues with an assignee

What about a `roadmap` label?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 5
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+# type/discussion - our discussion pages. They can be inactive for a long time.
+# WIP - pull requests that have "work in progress" status
+exemptLabels:
+  - type/discussion
+  - WIP
```
Comment (San4es) [inline .github/stale.yml:15]:
```suggestion
  - WIP
  - roadmap
```

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```
Comment (San4es) [inline .github/stale.yml]:
```suggestion
issues:
  closeComment: >
     This issue is closing due to inactivity.
     In the case of bugs or questions, feel free to create new GitHub [issues](https://github.com/DevExpress/DevExtreme/issues) or tickets in our [Support Center](https://www.devexpress.com/Support/Center/).
```

Comment (DokaRus) [reply .github/stale.yml]: LGTM

@RomanTsukanov  could you check it?

Comment (RomanTsukanov) [reply .github/stale.yml]:
```suggestion
issues:
  closeComment: >
     This issue was closed due to inactivity.
     If you found a bug or have a question about our product, [create a new issue](https://github.com/DevExpress/DevExtreme/issues) on GitHub or submit a ticket in our [Support Center](https://www.devexpress.com/Support/Center/).
```

**Review by RomanTsukanov — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Review by AntonSermyazhko — APPROVED** [review]

**Pull request merged**